app/frontend/store.js

```diff
@@ -30,6 +30,15 @@ const initialState = {
   validationMsg: undefined,
   authMethod: 'mnemonic',
   enableTrezor: CARDANOLITE_CONFIG.CARDANOLITE_ENABLE_TREZOR,
+  donateBtcUrl: CARDANOLITE_CONFIG.CARDANOLITE_DONATE_BTC_ADDRESS
+    ? `https://www.blockchain.com/btc/address/${CARDANOLITE_CONFIG.CARDANOLITE_DONATE_BTC_ADDRESS}`
```

Comment:
I would make this URL a named constant and put it in constants.js. Same for ETH.